src/SFA.DAS.Tasks.AcceptanceTests/Features/Transfers.feature

```diff
@@ -6,3 +6,20 @@ Scenario: 040 Sent Transfer Connection Invitation
 Given I have Sent Transfer Connection Invitation
 When transfer_connection_invitation_sent message get publish
 Then I should have a ReviewConnectionRequest Task
+
+
+Scenario: 041 Accept Transfer Connection Invitation
+Given I have Sent Transfer Connection Invitation
+When transfer_connection_invitation_sent message get publish
+Then I should have a ReviewConnectionRequest Task
+Given I have Approved A Transfer Connection Invitation
+When approved_transfer_connection_invitation message get publish
+Then I should not have a ReviewConnectionRequest Task
+
+Scenario: 042 Accept Transfer Connection Invitation
```

Comment:
Reject Transfer Connection Invitation?

Reply:
Odd as it was that name on my local (committed version), but I have pushed a new commit to nudge it and change the name now!